### DISCUSSION TITLE
SVG Optimization

### DISCUSSION THREAD

**Discussion by l-alexandrov** [spatie/laravel-image-optimizer#114] [Q&A]
I see that the underlying library [spatie/image-optimizer](https://github.com/spatie/image-optimizer) requires installing `svgo@1.3.2` which is very outdated. According to its `README.md` another `svgo` version could be used if you [create a custom optmiziation chain](https://github.com/spatie/image-optimizer/tree/main#svgs).

Do the defaults in the `config/image-optimizer`  create such chain and would it work with a newer `svgo` version?

**Comment by freekmurze** [comment]
Test it out, and if that works send a PR to update our docs! 👍
